VideoLocker/src/main/java/org/edx/mobile/view/adapters/InfiniteScrollUtils.java

```diff
@@ -100,8 +101,17 @@ private void onLoadMore() {
             adapter.setProgressVisible(true);
             pageLoader.loadNextPage(new PageLoadCallback<T>() {
                 @Override
+                public void onPartialPageLoaded(List<T> newItems) {
+                    if (isResetCalled()) {
+                        return; // Disregard result, since reset() was called
```

Comment:
Note that if the `isResetCalled()` method is renamed to `isAbandoned()` per my previous suggestion, then this comment can be changed to even removed (as it would be pretty self-explanatory). Same goes for the comment on the `onPageLoaded()` callback method.